src/app.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import '../styles/index.scss';
```

Comment:
since we have `ExtractTextPlugin` in production, the files is extruded anyway for build but watched for changes in dev mode